backend/api/Startup.cs

```diff
@@ -184,11 +177,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         /// <param name="app"></param>
         private static void UpdateDatabase(IApplicationBuilder app)
         {
-            using (var serviceScope = app.ApplicationServices
+            using(var serviceScope = app.ApplicationServices
```

Comment:
/nit: According to [C# documentation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement) there should be a space after "using".

Reply:
Hmm I'll see if vscode is causing me more pain here.